### PR TITLE
#332 add .editorconfig

### PR DESCRIPTION
**Pull request by AndreasAugustin**
Just added the `.editorconfig`.
The configuration is set like the current file types have their style.

No code changes or addition.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+
+[*.{ts,json,js}]
```
Comment (Rugvip) [inline .editorconfig]: Should include tsx and jsx as well right?

**Review by Rugvip — APPROVED** [review]
Awesome, thanks! :grin:

**Comment by freben** [comment]
Apart from missing ts/tsx, looks good!

File: .editorconfig (hunk 1 above)
Comment (AndreasAugustin) [reply .editorconfig]: Oh, forgot that. Thanks!

Comment (AndreasAugustin) [reply .editorconfig]: Also `*.html` was missing. Now it should do the job :+1:

**Review by freben — APPROVED** [review]

**Review by soapraj — APPROVED** [review]

**Pull request merged**